python/tests/unit/arcticdb/version_store/test_sort_merge.py

```diff
@@ -612,4 +614,41 @@ def test_update_symbol_list(lmdb_library):
     lib.write(sym_2, df, staged=True)
     lib.sort_and_finalize_staged_data(sym_2, mode=StagedDataFinalizeMethod.APPEND)
     assert lib_tool.count_keys(KeyType.SYMBOL_LIST) == 3
-    assert set(lib.list_symbols()) == set([sym, sym_2])
\ No newline at end of file
+    assert set(lib.list_symbols()) == set([sym, sym_2])
+
+class TestSlicing:
+    def test_long_append_segment(self, lmdb_library):
+        set_config_int('Merge.SegmentSize', 5)
```

Comment:
Use a context manager to ensure it is reset at the end of the test:
https://github.com/man-group/ArcticDB/blob/008fa54cfe123c9ac4276643ebb5e22126d7a3c0/python/arcticdb/util/test.py#L122